### ISSUE TITLE
Accessibility improvements and keyboard navigation issues for the Editor

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-ui-core#6811]
Reported by client (Ticket ID: 1561246)

1. NVDA narrator problems:
    - Doesn't narrate the disabled buttons

2. Tab Order issues:
    - Few of the dropdowns get open, but unable to access the content from keyboard (color picker and background picker for example)
    - After closing dropdown with Esc key, keyboard navigation is not working properly

3. Aquatic mode:
    - Toolbar is visible, but keyboard focus not is not visible on toolbar

**Comment by veselints** [comment]
Closing the above as:
1. I have created a dedicated issue for that: https://github.com/telerik/kendo-ui-core/issues/6813
2. The Color pickers focus issue has been fixed: https://github.com/telerik/kendo-ui-core/issues/6654.
3. There is no Aquatic mode of the Editor or a Theme with such a name

**Comment by Neeraj-apshankar** [comment]
> Closing the above as:
> 
> 1. I have created a dedicated issue for that: [Editor disabled buttons are not focusable #6813](https://github.com/telerik/kendo-ui-core/issues/6813)
> 2. The Color pickers focus issue has been fixed: [Unable to navigate between colors in Editor Font and Background ColorPickers with keyboard #6654](https://github.com/telerik/kendo-ui-core/issues/6654).
> 3. There is no Aquatic mode of the Editor or a Theme with such a name

Hello,
**_> 2. The Color pickers focus issue has been fixed:**
Can you please tell in which version of KendoEditor this is fixed?
_
